### PR TITLE
Support Xcode 7 Beta 3 and remove unused variable

### PR DESCRIPTION
**Pull request by mcudich**
One more small one.

### CODE REVIEW

File: SwiftBox/Layout+View.swift [hunk 1]
```diff
@@ -21,7 +21,7 @@ extension Layout {
 	public func apply(view: ViewType) {
 		view.frame = CGRectIntegral(frame)
 
-		for (s, layout) in Zip2(view.subviews, children) {
+		for (s, layout) in Zip2Sequence(view.subviews, children) {
```
Comment (joshaber) [inline SwiftBox/Layout+View.swift]: Can we just use the `zip` free function instead, so it works on Swift 1.2 and Swift 2?

File: Generated/Layout.c [hunk 2]
```diff
@@ -438,7 +438,6 @@ static void layoutNodeImpl(css_node_t *node, float parentMaxWidth) {
   // We want to execute the next two loops one per line with flex-wrap
   int startLine = 0;
   int endLine = 0;
-  int nextOffset = 0;
```
Comment (joshaber) [inline Generated/Layout.c]: Given that this is generated, we should probably just ignore all warnings on these files.

**Comment by joshaber** [comment]
Thanks! Just a couple comments.

File: Generated/Layout.c (hunk 2 above)
Comment (mcudich) [inline Generated/Layout.c]: Done.

File: SwiftBox/Layout+View.swift (hunk 1 above)
Comment (mcudich) [inline SwiftBox/Layout+View.swift]: Done!

**Comment by joshaber** [comment]
:metal:

**Pull request merged**